### ISSUE TITLE
right and left foot perfusion

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Vocabulary-v5.0#439]
For diabetes control we have a large number of measurements of right and left foot perfusion.

Any guidance where this should be mapped to is welcome.

p.s. do these questions belong here or somewhere else?

Thanks.

**Comment by cgreich** [comment]
What's the measurement, @PRijnbeek? Are you looking for ["Diabetic foot"](https://athena.ohdsi.org/search-terms/terms/4087682)? That is a Condition. There is not much of Measurements to determine that, it's a diagnosis based on signs and symptoms revealed through examination.

Or do you have some special tests like videomicroscopy?

**Comment by PRijnbeek** [comment]
:0 i know that.

It is as i state in the post. The **Measurement** of the perfusion in the foot. There is no condition here.

For example: https://www.researchgate.net/publication/283570256_Assessment_of_foot_perfusion_in_patients_with_diabetic_foot_ulcer

**Comment by cgreich** [comment]
Right. So, you got examination and interviewing the patient. No procedures or measurements for that. The measurements mentioned are:

- Transcutaneous oxygen tension: https://athena.ohdsi.org/search-terms/terms?domain=Measurement&standardConcept=Standard&page=1&pageSize=500&query=oxygen+%5Bpartial+pressure%5D%2A&boosts
- Ankle-brachial pressure index: https://athena.ohdsi.org/search-terms/terms/46237026
- Toe pressure index: https://athena.ohdsi.org/search-terms/terms/46236281

Cross-sectional imaging (x-ray, ct, mri, depending what you have), fluorescence angiography, capillaroscopy, iontophoresis and hyperspectral imaging are all **procedures**. Today we cannot process results of diagnostic procedures that are more complex and produce more than a simple value/unit result.

Bottom line: If you have a concrete measurement, put it in. Otherwise it is the result of complex decision making resulting in a diagnosis (=Condition).

**Comment by PRijnbeek** [comment]
Thanks i am expecting to get information about the unit of those "measurements" later today to answer your question.

**Comment by PRijnbeek** [comment]
its values are normal/abnormal/undetermined

**Comment by cgreich** [comment]
I would write the "Diabetic foot" Condition above if it is abnormal. If it is normal, or undetermined - drop it. Would that work?

**Comment by dimshitc** [comment]
Seems, perfusion should be drastically decreased for Diabetic foot to develop.
What if they want to reflect slight decrease of a perfusion?

**Comment by PRijnbeek** [comment]
Exactly that would be too strong. That would require more information to then say Diabetic foot based on that measurement alone.  For you background in the Netherlands these test are standard for any person that has the diabetes diagnosis. It is part of the care plan.

So it seems we do not have a good solution for this correct? We can keep it mapped to zero and use the source_concept_id for now. It would just bump our vocab mapping stats considerably and since we will be doing diabetes research it may become handy at some point.

**Comment by dimshitc** [comment]
@cgreich @Alexdavv what do you think of adding these Measurements to the OMOP Extension?

**Comment by MelaniePhilofsky** [comment]
Seems like this branch of SNOMED might be appropriate: https://athena.ohdsi.org/search-terms/terms/4161988

**Comment by dimshitc** [comment]
I can't see the exact perfusion of foot there.
Or do you mean that we can extend this branch by an OMOP Extension concepts?

**Comment by Alexdavv** [comment]
> Seems like this branch of SNOMED might be appropriate: https://athena.ohdsi.org/search-terms/terms/4161988 

As long as we don't know how and what exactly was measured, [this](https://athena.ohdsi.org/search-terms/terms/4095647) SNOMED Observable entity is a proper choice, I think.

There's [another one](https://athena.ohdsi.org/search-terms/terms/4088756) pre-coordinated Condition. But it seems we'll split and map them over in the future.

If there's a need to be more specific to make a distinction between leg/foot and right/left, a new OMOP Extension concept would be a choice.


> Bottom line: If you have a concrete measurement, put it in. Otherwise it is the result of complex decision making resulting in a diagnosis (=Condition).

I wouldn't be so strict about it. We have many abstractional Measurements here and there. And once SNOMED treat it as an observable entity, I would allow to use it with a value in the Measurement Domain. There's no a good way to differ them from the real Measurements anyway.

**Comment by cgreich** [comment]
> So it seems we do not have a good solution for this correct? 

@PRijnbeek, you need to find out what it is that they are testing. We have all the tests (see above). But a generic "no perfusion" is not a test, it is a conclusion=diagnosis. They must be testing something, and somebody will know.

> As long as we don't know how and what exactly was measured, this SNOMED Observable entity is a proper choice, I think.

We don't know if it is arterial or capillary.

> If there's a need to be more specific to make a distinction between leg/foot and right/left, a new OMOP Extension concept would be a choice

Agreed.

**Comment by p-talapova** [comment]
@PRijnbeek Just FYI. According to "The Best Practice Recommendations for the Prevention and Management of Peripheral Arterial Ulcers", for the assessment of foot perfusion physicians usually use [ankle-brachial pressure index (ABPI)](https://geekymedics.com/ankle-brachial-pressure-index-abpi-measurement-osce-guide/).
Resulsts can be interpreted as normal (1.0-1.2) and abnormal (mild arterial disease(0.8-0.9)/moderate arterial disease(0.5-0.79)/severe arterial disease(<0.5));

in OMOP, we have 
1) standard procedures (all in SNOMED): 
4295155	401221002	Ankle brachial pressure index 
44805248	751071000000105	Left ankle brachial pressure index
44805247	751061000000103	Right ankle brachial pressure index
4295156	401222009	Left dorsalis pedis ABPI
4268437	401224005	Left posterior tibial ABPI
4268436	401223004	Right dorsalis pedis ABPI
4268438	401225006	Right posterior tibial ABPI	

2) standard measurements: 
40489833	446841001	Ankle brachial pressure index (SNOMED)
46237026	77194-9	Ankle-brachial index (LOINC)

**Comment by PRijnbeek** [comment]
Thanks that is helpful.
